src/sdformat_urdf_plugin.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <rcutils/logging_macros.h>
+#include <tinyxml2.h>
+#include <urdf_parser_plugin/parser.h>
+
+#include <limits>
+#include <string>
+
+#include "sdformat_urdf/sdformat_urdf.hpp"
+
+namespace sdformat_urdf
+{
+class SDFormatURDFParser final : public urdf::URDFParser
+{
+public:
+  SDFormatURDFParser() = default;
+  ~SDFormatURDFParser() = default;
+
+  urdf::ModelInterfaceSharedPtr
+  parse(const std::string & data) override
+  {
+    urdf::ModelInterfaceSharedPtr urdf_cpp;
+    sdf::Errors errors;
+    urdf_cpp = sdformat_urdf::parse(data, errors);
```

Comment:
Nit: you could save a line of code here:

```suggestion
    sdf::Errors errors;
    urdf::ModelInterfaceSharedPtr urdf_cpp = sdformat_urdf::parse(data, errors);
```

Reply:
Saved the line of code in c9ca571